pkg/operator/etc/config.go

```diff
@@ -25,6 +25,7 @@ type Config struct {
 	VulnerabilityScannerEnabled                  bool           `env:"OPERATOR_VULNERABILITY_SCANNER_ENABLED" envDefault:"true"`
 	VulnerabilityScannerScanOnlyCurrentRevisions bool           `env:"OPERATOR_VULNERABILITY_SCANNER_SCAN_ONLY_CURRENT_REVISIONS" envDefault:"false"`
 	VulnerabilityScannerReportTTL                *time.Duration `env:"OPERATOR_VULNERABILITY_SCANNER_REPORT_TTL"`
+	ScanJobInSameNamespace                       bool           `env:"OPERATOR_ENABLE_SCAN_JOB_IN_SAME_NAMESPACE" envDefault:"false"`
```

Comment:
See other params related to vulnerability scanning:
```suggestion
	VulnerabilityScannerSameNamespace                       bool           `env:"OPERATOR_VULNERABILITY_SCANNER_SAME_NAMESPACE" envDefault:"false"`
```